frontend/src/js/actions/workspaces/deleteOrRemoveItem.ts

```diff
@@ -0,0 +1,31 @@
+import { deleteOrRemoveItem as deleteOrRemoveItemApi } from '../../services/WorkspaceApi';
+import { getWorkspace } from './getWorkspace';
+import { ThunkAction } from 'redux-thunk';
+import { AppAction, AppActionType, WorkspacesAction } from '../../types/redux/GiantActions';
+import { GiantState } from '../../types/redux/GiantState';
+
+export function deleteOrRemoveItem(
```

Comment:
should this have a different name as I think it is now just 'delete' which might fail with an error?

Reply:
good spot, yes I just changed the name to `deleteResourceFromWorkspace` does it sound OK?